mmocr/models/textrecog/layers/transformer_layer.py

```diff
@@ -283,12 +282,143 @@ def forward(self, x):
         return x
 
 
+class PositionAttention(nn.Module):
```

Comment:
This part can be split two-part: Position and Attention, which are declared in ABIVisionDecoder.